stock_storage_type/models/stock_storage_location_sequence_cond.py

```diff
@@ -112,13 +112,13 @@ def _exec_code(self, storage_location_sequence, putaway_location, quant, product
                 "Condition %s not met:\n"
                 "* putaway sequence: %s\n"
                 "* putaway location: %s\n"
-                "* quant: %s\n"
+                "* quants: %s\n"
                 "* product: %s\n"
                 % (
                     self.name,
                     storage_location_sequence.id,
                     putaway_location.name,
-                    quant.id,
+                    quant.ids,
```

Comment:
Should we rename this parameter everywhere with plural form?
At the beginning this parameter is passed this way BTW:
```python
            if not package_sequence.can_be_applied(putaway_location, quants, product):
                continue
```
And afterwards the singular form is used.